.github/workflows/dev-deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: klaytn-safe-react deploy workflow
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  deploy:
+    permissions:
+      id-token: write
+      contents: read
+    runs-on: ubuntu-latest
+    
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Get AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE_ARN_DEV }}
+          role-session-name: SessionForKlaytnActions
+          aws-region: ${{ secrets.AWS_REGION_DEV }}
+      - name: Installing Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "14"
+      - name: Installing dependencies
+        run: yarn install
+      - name: Change homepage and get .env

```

Comment:
After the **yarn install,** Could you also please write the copy commands for https://github.com/klaytn/klaytn-safe-react/tree/dev/safe-deployment-assets/v1.3.0 to move into the node_modules as mentioned in the Dockerfile.